apps/andi/mix.exs

```diff
@@ -50,8 +50,8 @@ defmodule Andi.MixProject do
       {:divo_kafka, "~> 0.1.5", only: [:dev, :integration]},
       {:divo_postgres, "~> 0.2", only: [:dev, :integration]},
       {:divo_redis, "~> 0.1.4", only: [:dev, :integration]},
-      {:ecto, "== 3.5.4"},
-      {:ecto_sql, "== 3.3.3"},
+      {:ecto, "== 3.9.0", override: true},
```

Comment:
ecto_sql requires a matching ecto, so :ecto_sql 3.6.2 requires :ecto 3.6.2

I tried to do what it wanted and bumped ecto_sql, which requires a new postgresx, which requires a new guardian_db, which requires a new postgrex, which requires a new elixir.....

Its pretty critical to have the new :ecto version. It has the force_changes: true flag for changesets, which is a game changer in design.

I feel the least egregious version discrepancy is a forced mismatch begin ecto and ecto_sql, rather than a forced mismatch of ecto_sql and postgres/guardian/elxir